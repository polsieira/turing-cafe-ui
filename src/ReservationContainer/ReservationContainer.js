import React from 'react';
import '../ReservationContainer/ReservationContainer.scss';
import ReservationCard from '../ReservationCard/ReservationCard';

const ReservationContainer = ({ reservations, deleteReservation }) => {

  const reservationCards = reservations.map(reservation => {
    return <ReservationCard
      id={reservation.id}
      name={reservation.name}
      key={reservation.id}
      date={reservation.date}
      time={reservation.time}
      numberOfGuests={reservation.number}
      deleteReservation={deleteReservation} />
  })

  return (
    <>
      {reservationCards}
    </>
  )
}

export default ReservationContainer;