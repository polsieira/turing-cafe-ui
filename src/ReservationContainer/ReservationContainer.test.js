import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import ReservationContainer from './ReservationContainer';

describe('ReservationContainer', () => {

  let wrapper;
  const mockDeleteReservation = jest.fn();
  const reservations = []

  beforeEach(() => {
    wrapper = shallow(<ReservationContainer deleteReservation={mockDeleteReservation} reservations={reservations} />)
  });

  it('should match a snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
});