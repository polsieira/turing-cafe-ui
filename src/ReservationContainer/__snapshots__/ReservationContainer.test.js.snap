// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReservationContainer should match a snapshot 1`] = `<Fragment />`;
